src/types/redis_string.cc

```diff
@@ -213,12 +213,12 @@ rocksdb::Status String::GetDel(const std::string &user_key, std::string *value)
 
 rocksdb::Status String::Set(const std::string &user_key, const std::string &value) {
   std::vector<StringPair> pairs{StringPair{user_key, value}};
-  return MSet(pairs, 0);
+  return MSet(pairs, 0, true);
```

Comment:
We'd better mark the argument:
```suggestion
  return MSet(pairs, /*ttl=*/0, /*lock=*/true);
```